cmd/terramate/cli/run.go

```diff
@@ -148,12 +159,25 @@ func (c *cli) runOnStacks() {
 		}
 	}
 
-	err = c.RunAll(runStacks, isSuccessExit)
+	err = c.RunAll(runs, isSuccessExit, RunAllOptions{
+		Quiet:           c.parsedArgs.Quiet,
+		DryRun:          c.parsedArgs.Run.DryRun,
+		ScriptRun:       false,
+		ContinueOnError: c.parsedArgs.Run.ContinueOnError,
+	})
 	if err != nil {
 		c.fatal("one or more commands failed", err)
 	}
 }
 
+// RunAllOptions define named flags for RunAll
+type RunAllOptions struct {
```

Comment:
We could make it package private since its only used internally?

```suggestion
// runAllOptions define named flags for RunAll
type runAllOptions struct {
```

Reply:
It's tied to `RunAll`, so we would have to make that private, too. You are not wrong, but I think most things in the cli package could be private. The function is now shared by `script run` and `run`, so you could say it's internal public as opposed to internal internal :)

IMO we should refactor this eventually, but for now I'd prefer to keep it as it is.

Reply:
We could make `RunAll` private too? I agree that there are other occurrences of functions that should be provided but since you are touching RunAll and adding the options it seems worth it to fix some low hanging fruit :)